### ISSUE TITLE
imagenet resnet pytorch bad performance on test set

### ISSUE THREAD

**Issue by priyakasimbeg** [mlcommons/algorithmic-efficiency#192]
The imagenet pytorch resnet workload does not show improving accuracy on the test set, suggesting a bug in the test data pipeline. 

## Description
The end of training, the test/accuracy was only 0.001.
```
{'train/accuracy': 0.89036, 'train/loss': 0.559258671875, 'validation/accuracy': 0.7096, 'validation/loss': 1.31008859375, \'test/accuracy': 0.001, 'test/loss': 7.7848640625, 'global_step': 97635}), (97966, {'train/accuracy': 0.87816, 'train/loss':   0.63010578125, 'validation/accuracy': 0.70652, 'validation/loss': 1.33117421875, 'test/accuracy': 0.001, 'test/loss': 7.6467, \'global_step': 97966}), (98299, {'train/accuracy': 0.87696, 'train/loss': 0.6126491796875, 'validation/accuracy': 0.70756, \'validation/loss': 1.3214384375, 'test/accuracy': 0.001, 'test/loss': 7.4925984375, 'global_step': 98299})], 'global_step': \98571}
I1015 18:09:22.268874 140444536985408 submission_runner.py:422] Timing: 111600.52427244186
I1015 18:09:22.268943 140444536985408 submission_runner.py:423] ====================
I1015 18:09:22.269364 140444536985408 submission_runner.py:483] Final imagenet_resnet score: 111600.524272
```
## Steps to Reproduce
On kasimbeg-5 vm run:
```
submission_runner.py \
    --framework=pytorch \
    --data_dir=~/local_data/imagenet/pytorch \
    --imagenet_v2_data_dir=~ \
    --experiment_dir=~ \
    --experiment_name=target_setting \
    --workload=imagenet_resnet \
    --submission_path=reference_algorithms/target_setting_algorithms/pytorch_nesterov.py \
    --tuning_search_space=reference_algorithms/target_setting_algorithms/imagenet_resnet/tuning_search_space.json > results_imagenet_resnet_pytorch.txt 2>&1
```

## Source or Possible Fix
Since the other metrics improved on train and validation data sets, I think there is a bug in the pipeline for the test set.
